### PR TITLE
Feature/impl method remaining courses

### PR DESCRIPTION
**Pull request by morenocami**
Function is not in the proper place, but it will be moved once we work on the corresponding controller.

I tested it with console output and it works as expected.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+
+
   def index
+    @username = params["uname"]
+    @password = params["pwd"]
+
+    u = User.where(:username => @username).first
+
+    if u == nil
+      puts "Error"
```
Comment (jbitton) [inline app/controllers/home_controller.rb]: Why are you using `puts` over here and for the other if statements? you cannot `puts` to a web application; only to the console. if you wanted to output something to the console, then use `logger.info`.  Anyway, you shouldn't even do that. You should possibly use `flash.now` or some other method to create a pop up

File: app/views/home/index.html.haml [hunk 2]
```diff
@@ -40,16 +44,16 @@
         %section
           .content
             .inner
-              %h2 Please enter your FAUNetID and Password
-              %form
+              %h2 Please enter your Username and Password
+              %form{:action => home_index_path}
```
Comment (jbitton) [inline app/views/home/index.html.haml]: why are you doing this?

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (morenocami) [reply app/controllers/home_controller.rb]: Great, I'll check that out. Puts did appear on console so I used it then. But I'll use flash.now from now  on.

File: app/views/home/index.html.haml (hunk 2 above)
Comment (morenocami) [reply app/views/home/index.html.haml]: I'll take a second look at it. Maybe it's something from my failed attempts that I forgot to remove :pI'll take a second look at it. Maybe it's something from my failed attempts that I forgot to remove :p

**Review by APerlman2015 — APPROVED** [review]
looks good!

**Pull request merged**